content/Foundation/Summit/2024/home.pml

```diff
@@ -0,0 +1,93 @@
+<!--<title>Summit</title>-->
+<!--
+    Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+<h1>
+  seL4 Summit 2024
+</h1>
+<h4>
+  Sydney, Australia
+</h4>
+<h4>
+  October 2024 (dates TBC)
+</h4>
+
+
+<div class="summit_div">
+  <ul>
+    <!--li><a href="#summit-important-dates">Important Dates</a></li-->
+    <li><a href="#summit-about">About</a></li>
+    <li><a href="../principles">Principles</a></li>
+    <!--li><a href="#summit-registration">Registration</a></li-->
+    <li><a class="highlight" href="#latest-news">Latest News: Location & Venue</a></li>
+    <!--li><a href="cfp">Call for Presentations</a></li>
+    <li><a href="#summit-keynote-speakers">Keynote Speakers</a></li>
+    <li><a href="#summit-panellists">Panellists</a></li>
+    <li><a href="#summit-program">Program, Abstracts, Slides and Videos</a></li>
+    <li><a href="#summit-pc">Program Committee</a></li>
+    <li><a href="#summit-sponsors">Sponsors</a></li>
+    <li><a href="#summit-contact">Contact</a></li-->
+  </ul>
+</div>
+
+<div class="summit_div">
+  <a href="../">All summits</a>
```

Comment:
The reason we changed to "All summits" was to avoid having to go back and add a year list (with links) to the previous years' summit pages. Having said that it's not a lot of work, so I've added the years with links.